libraries/webcheckout/subscriptions/index.js

```diff
@@ -30,6 +32,15 @@ export default function shopify(subscribe) {
     dispatch(redirectRoute());
   });
 
+  /**
+   * Removes the login page if the user logged-in in the web checkout
+   */
+  subscribe(appDidStart$, ({ dispatch }) => {
```

Comment:
Pleqse subscribe to another stream

subscribe(userDidLogin$, ({ dispatch }) => {
      dispatch(redirectRoute())
 })